### DISCUSSION TITLE
How to fix this

### DISCUSSION THREAD

**Discussion by BeepPlayz** [rapid7/metasploit-framework#16085] [Q&A]
Traceback (most recent call last):
        16: from /data/data/com.termux/files/usr/bin/msfconsole:17:in `<main>'
        15: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
        14: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
        13: from /data/data/com.termux/files/home/metasploit-framework/config/boot.rb:26:in `<top (required)>'
        12: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
        11: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
        10: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler/setup.rb:10:in `<top (required)>'
         9: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler/ui/shell.rb:88:in `silence'
         8: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler/ui/shell.rb:136:in `with_level'
         7: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler/setup.rb:10:in `block in <top (required)>'
         6: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler.rb:149:in `setup'
         5: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler/runtime.rb:22:in `setup'
         4: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler/shared_helpers.rb:76:in `set_bundle_environment'
         3: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler/shared_helpers.rb:297:in `set_bundle_variables'
         2: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/bundler/rubygems_integration.rb:223:in `bin_path'
         1: from /data/data/com.termux/files/usr/lib/ruby/2.7.0/rubygems.rb:254:in `bin_path'
/data/data/com.termux/files/usr/lib/ruby/2.7.0/rubygems.rb:277:in `find_spec_for_exe': Could not find 'bundler' (2.3.5) required by your /data/data/com.termux/files/home/metasploit-framework/Gemfile.lock. (Gem::GemNotFoundException)
To update to the latest version installed on your system, run `bundle update --bundler`.
To install the missing version, run `gem install bundler:2.3.5`

**Comment by bcoles** [comment]
> Could not find 'bundler'

`bundler` is not installed for your version of Ruby or the version of `bundler` is too old.

> To update to the latest version installed on your system, run bundle update --bundler

To update `bundler`, run `bundle update --bundler`.

> To install the missing version, run gem install bundler:2.3.5`

To install `bundler`, run `gem install bundler:2.3.5`.

**Comment by 0FFSTYLE** [comment]
Install bundler-2.3.5  then re&run metasploit installer
If you still get same error install kali-nethunter rootless in your device

Kali-nethunter : https://www.kali.org/docs/nethunter/nethunter-rootless/

Termux no more giving updates on Play Store. So if you're using old version of termux install latest version from f-droid
